samples/ambient-argo/README.md

```diff
@@ -0,0 +1,99 @@
+# Ambient Reference Architecture w/ Argo
+
+This repo contains a reference architecture for operating Istio Ambient Mesh with ArgoCD using GitOps.  It demonstrates best practices for leveraging Istio as part of an application platform.
+
+## :boom: DISCLAIMER
+
+Istio Ambient Mesh is still in Alpha, and is not suitable for production use.  Likewise, this reference architecture is of Alpha quality, and includes several rough edges, including:
+ * Cluster-Scoped upgrades cause known traffic loss, and have wide blast radius.
+ * The tag chart is forked from the primary istio repo, and needs to be merged and published
+ * CRDs are not currently upgraded
+
+## Getting Started
+
+This reference architecture assumes that you have an ArgoCD installation with:
+ * A [connected cluster](https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_cluster/) named `ambient-cluster`
+ * A connection to your repository (for private repos)
+
+To deploy Istio, supporting software, and the bookinfo sample application, copy this folder to the root of your repo and run:
+
+```bash
+read -p 'Please enter the URL to your repo:'
+OLD_REPO='{repo-placeholder}'
+find . \( -type d -name .git -prune \) -o -type f -name '*.yaml' -print0 | xargs -0 sed -i s,$OLD_REPO,$NEW_REPO,g
+argocd create application -f meta-application.json
```

Comment:
Even after reading all this I'm still not clear on what's happening with the meta application thing, but I'm also not very familiar with Argo

Reply:
meta-application tells argocd to sync all yaml files in the istio/ folder, and application/application.yaml.